spacewalk/setup/lib/Spacewalk/Setup.pm

```diff
@@ -1119,10 +1149,19 @@ sub postgresql_clear_db {
 
 sub postgresql_drop_reportdb {
         my $answers = shift;
-
-        system_debug('/usr/bin/uyuni-setup-reportdb', 'remove',
-            '--db='.$answers->{'report-db-name'}, '--user='.$answers->{'report-db-user'});
-        return 1;
+        if ($answers->{'externaldb'}) {
+          system_debug('/usr/bin/uyuni-setup-reportdb', 'remove',
+              '--db='.$answers->{'report-db-name'}, '--user='.$answers->{'report-db-user'},
+                    "--externaldb-host", $answers->{'report-db-host'},
+                    "--externaldb-user", $answers->{'externaldb-admin-user'},
+                    "--externaldb-password", $answers->{'externaldb-admin-password'});
```

Comment:
externaldb-password should be externaldb-admin-password

Reply:
already fixed